security/org-account-assume/main.tf

```diff
@@ -101,56 +91,3 @@ resource "aws_iam_role_policy" "prime-admin" {
   provider = aws.workload
 }
 
-# Create cloud-engineer role for the workload account
-resource "aws_iam_role" "cloudengineer_role" {
-  name                 = var.cloudengineer_iam_role_name
-  description          = "Cloud-engineer role"
-  assume_role_policy   = module.iam_idp.adfs_assume_policy
-  max_session_duration = 28800
-
-  provider = aws.workload
-}
-
-# Policy inline to cloud-engineer role
-resource "aws_iam_role_policy" "cloudengineer" {
-  name   = "CloudEngineer"
-  role   = aws_iam_role.cloudengineer_role.id
-  policy = module.iam_policies.cloudengineer
```

Comment:
Is this policy used anywhere else? Maybe it can be cleaned up also.

Reply:
Yeah they are, can't remove it until other PR's are through